index.html

```diff
@@ -0,0 +1,291 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <link
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+      crossorigin="anonymous"
+    />
+    <title>Portofolio</title>
+  </head>
+  <body>
+    <header id="head-nav">
+      <h1 id="logo">
+        <a href=""> Welcome </a>
+      </h1>
+      <nav class="navigation">
+        <a id="bar" href="">
+          <i class="far fa-bars"></i>
+        </a>
+      </nav>
+    </header>
+    <section id="welcome-section">
+      <img
+        class="subHead"
+        src="./images/header-illsutration-mobile@2x.png"
+        alt=""
+      />
+      <div class="center">
+        <h2 class="name">Hey there. I'm Zachee</h2>
+        <h3 class="title">I am a software developer</h3>
+        <p class="description">
+          I can help you build a product, feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don't hestate to Contact me.
+        </p>
+        <div class="icons">
+          <ul class="font-icons">
+            <li>
+              <a href="">
+                <i class="fab fa-github-square"></i>
+              </a>
+            </li>
+            <li>
+              <a href="">
+                <i class="fab fa-linkedin-in"></i>
+              </a>
+            </li>
+            <li>
+              <a href="">
+                <i class="far fa-hand-peace"></i>
+              </a>
+            </li>
+            <li>
+              <a href="">
+                <i class="fab fa-twitter"></i>
+              </a>
+            </li>
+            <li>
+              <a href="">
+                <i class="fab fa-medium-m"></i>
+              </a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </section>
+    <div class="disable">
+      <img src="./images/Disabled.svg" alt="" />
+    </div>
+    <h1 id="first">My Recent</h1>
+    <h1 id="sec">Works</h1>
+    <hr />
+    <section class="project-section">
+      <ul id="project-cards">
```

Comment:
- [ ] Let's use the `id="projects-card` in the section tag instead of inside the ul. This is a unique identifier in your HTML page and it will add a reference to your entire section so that it can be accessed directly by an anchor link tag.